api/v1beta1/disruption_webhook.go

```diff
@@ -100,6 +102,17 @@ func (r *Disruption) ValidateCreate() error {
 		return fmt.Errorf("invalid disruption name: %w", err)
 	}
 
+	if specifiedEnvironment != "" {
+		specifiedEnv := r.Spec.SpecifiedEnvironment
+		if len(specifiedEnv) == 0 {
```

Comment:
can a string != "" be empty in Go ?

Reply:
the string is by default == "" if it's not set. It can be nil if the string is a pointer (specifically the pointer will be nil), but only in this case.